modules/notifications/assets/js/components/editor-drawer.js

```diff
@@ -0,0 +1,24 @@
+import { useEffect, useState } from 'react';
+import { WhatsNew } from './whats-new';
+
+export const EditorDrawer = () => {
+	const [ isOpen, setIsOpen ] = useState( true );
+	const [ isRead, setIsRead ] = useState( false );
+
+	useEffect( () => {
+		elementor.on( 'elementor/editor/panel/whats-new/clicked', () => setIsOpen( true ) );
+	}, [] );
+
+	useEffect( () => {
+		document.body.classList.remove( 'e-has-notification' );
+	}, [ isRead ] );
+
+	return (
+		<WhatsNew
+			isOpen={ isOpen }
+			setIsOpen={ setIsOpen }
+			setIsRead={ setIsRead }
```

Comment:
Why do you need it reactive? (if you also don't read the isRead value nowhere), instead you can just use it as an event and remove the isRead state:
```suggestion
			markAsRead={ () =>  document.body.classList.remove( 'e-has-notification' ) }
```